config/sidekiq.yml

```diff
@@ -0,0 +1,5 @@
+---
+:concurrency: 5
+:queues:
+  - default
+  - mailers
```

Comment:
Don't think we have this queue yet here, shall we just have the `default` queue for now?

Also, missing newline! 😛